examples/serial/serial_server.c

```diff
@@ -18,14 +21,13 @@ uintptr_t tx_active;
 uintptr_t rx_data;
 uintptr_t tx_data;
 
-struct serial_server global_serial_server = {0};
+serial_queue_handle_t rx_queue;
+serial_queue_handle_t tx_queue;
```

Comment:
This example code is supposed to be standalone and perhaps the naming of this file/protection domain is a bit misleading. It's not really a "server" as in it does not have clients.

If it was a component or library intended to be linked with code provided by users of sDDF then it would make sense to have all globals as `static`.

Reply:
Even then it should be static: It's instantly clear this variable is not accessed anywhere else. If the project gets bigger and there are multiple files, this becomes more important. Furthermore, it's a good habit to develop.

Only reason not to make it static is to ease debugging during live debugging sessions where the variable gets out-of-scope when you're in another file context.